.github/workflows/task-get-linux-configurations.yml

```diff
@@ -64,7 +64,10 @@ jobs:
           def needs_arch(arch):
             return requested_arch in ['all', arch]
 
-          naming = {'mariner:2': 'mcr.microsoft.com/cbl-mariner/base/core:2.0'}
+          naming = {
+            'mariner:2': 'mcr.microsoft.com/cbl-mariner/base/core:2.0',
+            'debian:bookworm': 'gcc:12.4.0-bookworm'
```

Comment:
I think we should track `gcc:12.4-bookworm` or `gcc:12-bookworm`

Reply:
Agreed, `12.4` in my opinion.